testutils/clustermanager/gke.go

```diff
@@ -295,69 +220,35 @@ func (gc *GKECluster) Acquire() error {
 		}
 	}
 	var cluster *container.Cluster
-	var op *container.Operation
+	var rb *container.CreateClusterRequest
+	rb, err = gke.NewCreateClusterRequest(request)
+	if err != nil {
+		return fmt.Errorf("failed building the CreateClusterRequest: '%v'", err)
+	}
 	for i, region := range regions {
 		// Restore innocence
 		err = nil
-		rb := &container.CreateClusterRequest{
-			Cluster: &container.Cluster{
-				NodePools: []*container.NodePool{
-					{
-						Name: "default-pool",
-						Autoscaling: &container.NodePoolAutoscaling{
-							Enabled:      true,
-							MinNodeCount: gc.Request.MinNodes,
-							MaxNodeCount: gc.Request.MaxNodes,
-						},
-						Config: &container.NodeConfig{
-							MachineType: gc.Request.NodeType,
-						},
-					},
-				},
-				Name: clusterName,
-				// The default cluster version is not latest, has to explicitly
-				// set it as "latest"
-				InitialClusterVersion: defaultGKEVersion,
-				// Installing addons after cluster creation takes at least 5
-				// minutes, so install addons as part of cluster creation, which
-				// doesn't seem to add much time on top of cluster creation
-				AddonsConfig: gc.getAddonsConfig(),
-				// Equivalent to --enable-basic-auth, so that user:pass can be
-				// later on retrieved for setting up cluster roles. Use the
-				// default username from gcloud command, the password will be
-				// automatically generated by GKE SDK
-				MasterAuth: &container.MasterAuth{Username: "admin"},
-			},
-			ProjectId: *gc.Project,
-		}
-
-		clusterLoc := getClusterLocation(region, gc.Request.Zone)
 
+		clusterLoc := gke.GetClusterLocation(region, request.Zone)
```

Comment:
I have been thinking about passing `region` and `zone` to all public APIs in `gke` instead of `loc`, WDYT?